webpack/assets/javascripts/react_app/components/SearchBar/__tests__/__snapshots__/integration.test.js.snap

```diff
@@ -47,9 +51,9 @@ Object {
     Array [
       Object {
         "payload": Object {
-          "query": "result",
+          "id": "bookmarksModal",
```

Comment:
Doesn't test if the query was updated, maybe change `takeStoreAndLastActionSnapshot` to  `takeActionsSnapshot`?

Reply:
@MariaAga good idea, updating.